package.json

```diff
@@ -26,16 +26,30 @@
   ],
   "eslintConfig": {
     "extends": "terra",
-    "rules": {
-      "react/forbid-prop-types": "off",
-      "import/no-extraneous-dependencies": [
-        "error",
-        {
-          "devDependencies": [
-            "./packages/*/src/terra-dev-site/**/*.jsx"
-          ]
+    "overrides": [
+      {
+        "files": [
+          "**/*/terra-dev-site/**/*.jsx"
```

Comment:
are we not disabling this rules for components and  jest tests .. ?

Reply:
No only examples and test files.